src/tree/localProject/LocalFunctionsTreeItem.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { AzureParentTreeItem, AzureTreeItem, createTreeItemsWithErrorHandling } from 'vscode-azureextensionui';
+import { functionJsonFileName } from '../../constants';
+import { FunctionConfig } from '../../FunctionConfig';
+import { localize } from '../../localize';
+import { nodeUtils } from '../../utils/nodeUtils';
+import { IProjectRoot } from './IProjectRoot';
+import { LocalFunctionTreeItem } from './LocalFunctionTreeItem';
+
+export class LocalFunctionsTreeItem extends AzureParentTreeItem<IProjectRoot> {
```

Comment:
Were you going to create a different superclass for these `Local...TreeItems`?

Reply:
Ideally yes - that was related to our discussion about `TreeItemPlus`. But it's too much work/refactoring that I don't want to do it before build